Jenkinsfile

```diff
@@ -46,32 +46,31 @@ pipeline
         {
             steps
             {
-                script
-                {
-                    echo "Run all the unittests"
-                    sh 'pipenv run invoke unittest'
-                    // TODO: call coverage report + save it
-                }
+                sh 'pipenv run pytest --junitxml=reports/testReport.xml --ignore tests/test_cc_pcan_can.py'
```

Comment:
I do not think we need the pcan_can information here.

Reply:
this part is for another york item